### ISSUE TITLE
Second unknown wet market theory

### ISSUE THREAD

**Issue by alexharv074** [Project-Evidence/project-evidence.github.io#29]
Section 8 has:

> At this time, it is widely believed that Patient Zero was infected in Wuhan, China with SARS-CoV-2. What is not so certain is where exactly in Wuhan Patient Zero was infected, and how he/she was infected. The competing theories are:
> 
> - From eating wild animals at the Huanan Seafood Market
>
> - From doing work at the Wuhan Institute of Virology, which performed gain-of-function research on the original SARS virus
>
> - From doing work at the Wuhan Centre for Disease Control, which performed experiments on and housed many bats known for carrying coronaviruses
>
> - From somewhere elsewhere entirely; they were not in Wuhan at all

How do we know that the virus did not come from a wet market in Wuhan other than the Huanan Seafood Market? This needs to be clarified for this document to be convincing.

**Comment by wisefish99** [comment]
How do we know the virus didn't come from a grocery store? How do we know the virus didn't come from aliens that dropped it off in Wuhan?

Theories that don't currently have any evidence pointing to them don't need to be addressed.

**Comment by alexharv074** [comment]
@wisefish99 I disagree. You have presented 4 possibilities and argue by eliminating the possibilities that it came from the Huanan market and then the possibility that it came from outside of Wuhan that it must have come from a lab. Not discussing the simplest hypothesis that the virus originated in Wuhan somewhere else is a major weakness that needs to be addressed.

**Comment by wisefish99** [comment]
Mmm, it's not my presentation. 

Perhaps the final point could be split up to say: 

- From somewhere else in Wuhan
- From somewhere outside of Wuhan entirely

This would be more collectively exhaustive, but would also establish the "from somewhere else in Wuhan" option as being on par with the other options, which I don't think it is. While it may be a simple hypothesis, it's also an unsupported one. Such discussion would be almost entirely conjecture instead of fact-based, antithetical to the very purpose of this analysis. 

As for the "from somewhere outside of Wuhan option", this section is somewhat lacking in the full Project E report I think, and does have credible and recent scientific work adding to its relevancy. I would like to see it expanded to at least include this work from a few weeks ago, where a UK team discuss three major strains of the virus in the world: 

Epidemiologist Dr. Peter Forster from Cambridge in the UK has theorized that the oldest virus strain (Type A) didn't originate in Wuhan. Type B, which began spreading in Wuhan, seems to be potentially more virulent and was detected more quickly. He believes it's far more likely to have originated in Guangdong, with some cases spreading to USA and Australia as Type A, and then later blowing up in Wuhan, as Type B, and then later spreading in Singapore and Europe as Type C. 

https://www.scmp.com/news/china/science/article/3080380/coronavirus-outbreak-may-have-started-september-say-british

**“If I am pressed for an answer, I would say the original spread started more likely in southern China than in Wuhan,” Forster said.** 

This seems an important quote from a respected Cambridge doctor in the UK, and his work should be included in the report I think. But that would hurt the overall narrative that the authors have spent so much effort trying to establish: that it was accidentally released in Wuhan. 

Here's an interview of Dr. Forster (on CGTN which obviously has its own biases, but I don't see why the Dr. himself would be biased). 

https://www.youtube.com/watch?v=AQQf2yoymu0

I seem to remember seeing somewhere that the majority of cases in New York are also a different strain than the rest of the country, suggesting more New York infections came from Europe imports, rather than East Asia. I will look more.

**Comment by Project-Evidence** [comment]
@alexharv074 The main argument against a second wet market or grocery store is that, if the infection began elsewhere, it would be simple to determine through contact tracing which location was mutual amongst sick patients. This is the same method that was used to focus on the Huanan Seafood Market in the first place. It would be unlikely for the Chinese CDC to single out this specific market if they had data arguing for another location in Wuhan. After all, why wouldn't they want to pick a market further from the labs? The outbreak theory would certainly be an outlier if the two closest labs were dozens of miles away from the market they picked. 

The only logical reason why they would deliberately mislead the public and pick the Huanan Seafood Market when their contact tracing pointed to another location would be if it pointed to the labs (assuming their data is accurate).

@wisefish99 

> From somewhere outside of Wuhan entirely

At the time of Revision 1 in early April, most news/scientific articles accepted the proposition that it began in Wuhan. The only major disagreement seemed to be among Chinese officials who claimed the virus was deployed by the U.S. (see 11.2 Lijian Zhao vs. Cui Tiankai). We included this section for completeness; although in our opinion it is obvious that these claims are more politically charged than based on scientific consensus.

If you are correct that the scientific community has moved towards considering locations outside of Wuhan, that deserves to be included in our report. One important section that would have to be addressed is 8.7 Outside of Wuhan - why didn't the first hospitalizations occur in, for example, southern China, as opposed to in Wuhan? Many people would have interacted with the infected animal. The first reported case of SARS-CoV-1, for example, was a farmer in a remote town - which makes sense for a natural outbreak:

> The SARS epidemic appears to have started in Guangdong Province, China, in November 2002 where the first case was reported that same month. The patient, a farmer from Shunde, Foshan, Guangdong, was treated in the First People's Hospital of Foshan.

https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome#Outbreak_in_South_China

> But that would hurt the overall narrative that the authors have spent so much effort trying to establish: that it was accidentally released in Wuhan.

We would not mind the inclusion of this information in the report. We have a counter-theory section, after all. 

We can include Dr. Forster's comments in the next revision. If you would like, feel free to find more sources theorizing the virus started outside of Wuhan, we would happily include them. :)

**Comment by wisefish99** [comment]
Here's the link for Dr. Forster's work. He's a forensic geneticist, not an epidemiologist, as I incorrectly stated previously. 

https://www.pnas.org/content/117/17/9241

Frankly I find it astounding that this didn't get more media coverage when it was published. Perhaps it was too "science-y", or perhaps there's no interest in hearing that Wuhan might not be the site for the "patient 0" spillover event. Anyway, here's the SparkNotes: 


- By using something called a "phylogenetic network" of 160 virus genomes, the researchers identified three predominant "clusters" of SARS-Cov-2 variants, which they dub Type A, B, and C.

- According to the mutation history, A is determined to be the "oldest" variant, with B having derived from A and C having derived from B.

- A is genetically most similar to the bat coronavirus found in China and is considered the "root" of the network. Nearly half the 33 cases in this A grouping are from outside East Asia, and only a handful are from Wuhan. 

- B is the most common variant found in Wuhan (about 75% of cases), and it appears to be mostly limited to people who were infected in Wuhan, Chinese and non-Chinese alike. 

- C is almost entirely absent from China, and is the predominant type in Europe. The researchers were able to trace one infection chain of Type C from Wuhan->Shanghai->Germany->Italy->Brazil.

The researchers don't offer conclusions about the site of "patient 0" in the study; Forster made his quote in the media. 

A major challenge I imagine has been the lack of complete genomic data from the earliest weeks of the outbreak. Type A appears to be less virulent than Types B or C, which may have led to its wider spreading for a longer period of time without detection, or misdiagnosis as influenza amidst the annual Chinese flu season. I wouldn't be shocked if a future analysis of flu deaths in China in Guangdong Province from October November, and December turns up instances of previously-undetected COVID-19 (with the Type A variant). That's one theory to explain fewer hospitalizations in Guangdong and its relative lack of detection until it mutated into the more aggressive and detectable Type B, with a super-spreader event at the Huanan market. 

Obviously 33 Type A cases in the study is not statistically sufficient and more data are needed. I suppose/hope Chinese health officials and researchers are looking at this now...

I'd also be very curious what variant the asymptomatic carriers generally have. My money's on A, but that's obviously speculation. 

I've seen speculation somewhere else, although I don't remember where and can't find it again, that it would be possible to simultaneously be infected with two variants. This would perhaps explain why patients seemingly recover and test negative, and then a few days later test positive again.

A somewhat related story: my friend who flew back to China from New York spent 3 weeks in a Shanghai quarantine after testing positive when she got off the plane. She tested negative twice, and then got ANOTHER positive hit and had to spend another week in Shanghai before testing negative and being able to leave. When she finally was able to fly to Shenzhen, she got ANOTHER positive test on her check-up and had to self-quarantine for another two weeks. Weird stuff...
